File: modul-06/PRAK601-2410817310014-INDRA_SURYADILAGA.py
matrik_baris = [[], []]
baris, kolom = map(int, input().split())
input_data = list(map(int, input().split()))

for i in range(0, len(input_data), kolom):
    baris = input_data[i:i + kolom]
    matrik_baris.append(baris)

for baris in matrik_baris:
    print(*baris)

File: modul-06/PRAK601-2410817310014-INDRA_SURYADILAGA.c
#include <stdio.h> 

int main() {
    int baris , kolom;
    int matrik_baris[100][100];
    scanf("%d %d", &baris, &kolom);

    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            scanf("%d", &matrik_baris[i][j]);
        }
    }
    
printf("\n");
    for (int i = 0; i < baris; i++) {
        for (int j = 0; j < kolom; j++) {
            printf("%d ", matrik_baris[i][j]);
        }
        printf("\n");
    }

    return 0;
}

